tools/lib/capitalization.py

```diff
@@ -77,6 +77,7 @@
      '<a href="/integrations" target="_blank">Integrations</a> or '
      '<a href="/api" target="_blank">API</a> documentation?'),
     r'Most stream administration is done on the <a href="/#streams">Streams page</a>.',
+    r'<h3>Your groups</h3><p>You are not a part of any groups.</p',
```

Comment:
This is wrong -- we should instead break that into 2 translatable strings, one for each block.

Reply:
Done!

Reply:
I am also thinking of refactoring this with work for issue #8802, it will make the whole thing clean!